tests/puppeteer/test/smoke/crawlingBO.js

```diff
@@ -0,0 +1,61 @@
+// importing pages
+const BO_LOGIN_PAGE = require('../../pages/BO/BO_login');
+const BO_DASHBOARD_PAGE = require('../../pages/BO/BO_dashboard');
+const BO_COMMON_PAGE = require('../../pages/BO/BO_commonPage');
+const BO_ORDERS_PAGE = require('../../pages/BO/BO_orders');
+const BO_PRODUCTS_PAGE = require('../../pages/BO/BO_products');
+const BO_CUSTOMERS_PAGE = require('../../pages/BO/BO_customers');
+
+let page;
+let BO_LOGIN;
+let BO_DASHBOARD;
+let BO_COMMON;
+let BO_ORDERS;
+let BO_PRODUCTS;
+let BO_CUSTOMERS;
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  BO_LOGIN = await (new BO_LOGIN_PAGE(page));
+  BO_DASHBOARD = await (new BO_DASHBOARD_PAGE(page));
+  BO_COMMON = await (new BO_COMMON_PAGE(page));
+  BO_ORDERS = await (new BO_ORDERS_PAGE(page));
+  BO_PRODUCTS = await (new BO_PRODUCTS_PAGE(page));
+  BO_CUSTOMERS = await (new BO_CUSTOMERS_PAGE(page));
+};
+
+// connect to the BO
+global.scenario('Crawl into BO et check a few key pages', async () => {
+  test('should login into BO', async () => {
```

Comment:
Please complete description of the scenario in comment

Reply:
Done thanks